package.json

```
@@ -81,7 +81,8 @@
     "cypress:run": "cypress run --e2e",
     "storybook": "start-storybook -p 6006 -s public",
     "build-storybook": "build-storybook -s public",
-    "chromatic": "chromatic --exit-zero-on-changes"
+    "chromatic": "chromatic --exit-zero-on-changes",
+    "ampli:pull": "ampli pull web"
```

Maybe we can add prettier formatting here?
```suggestion
    "ampli:pull": "ampli pull web & prettier ./src/index.ts"
```